sqlalchemy_ibmi/base.py

```diff
@@ -772,13 +772,20 @@ def create_connect_args(self, url):
         opts = url.translate_connect_args(username='user', host='system')
         opts.update(url.query)
         allowed_opts = {'system', 'user', 'password',
-                        'autocommit', 'readonly', 'timeout', 'database'}
+                        'autocommit', 'readonly', 'timeout',
+                        'database', 'library_list', 'current_schema'}
         if allowed_opts < opts.keys():
             raise ValueError("Option entered not valid for "
                              "IBM i Access ODBC Driver")
- 
-        return [["Driver={%s}; UNICODESQL=1; TRUEAUTOCOMMIT=1" % (
-                 self.pyodbc_driver_name)], opts]
+
+        connection_string = "Driver={%s}; UNICODESQL=1; TRUEAUTOCOMMIT=1;" % (
+                 self.pyodbc_driver_name)
+        libl = 'DefaultLibraries={},'.format(opts.get('current_schema', ''))
+        libl += ','.join(opts.get('library_list', ''))
+        if libl[-1] != '=':
```

Comment:
This won't work now that we always set DefaultLibraries. I'd suggest checking that current_schema or library_list were in opts instead.